gr-dtv/lib/dvbt2/dvbt2_p1insertion_cc_impl.cc

```diff
@@ -132,7 +133,7 @@ namespace gr {
       for (int i = 0; i < 384; i++) {
         dbpsk_modulation_sequence[i] = dbpsk_modulation_sequence[i + 1] * p1_randomize[i];
       }
-      memset(&p1_freq[0], 0, sizeof(gr_complex) * 1024);
+      std::fill_n(&p1_freq[0], 1024, 0);
```

Comment:
`fill_n` is a C++ construct. 3.7 branch will be stuck in C++03 land forever, probably.

Reply:
I believe fill_n() exists in C++98. Does it not?

Reply:
`std::fill_n` is already available prior to `C++11` it only changes it's output signature from `void` -> `OutputIterator`.

Reply:
Okay. If @mbr0wn agrees, I'll close this PR and open a new one with nicely separated commits.

Reply:
I misread cppreference.com. Yeah, please split (in case we need to revert individual changes or something) and we can merge. Thanks again!